config/bootstrap.php

```diff
@@ -64,6 +64,20 @@
 use Cake\Utility\Inflector;
 use Cake\Utility\Security;
 
+/**
+ * Read .env file if APP_NAME is not set.
+ *
+ * You can remove this block if you do not want to use environment
+ * variables for configuration when deploying.
```

Comment:
What do you think of 

> Environment variables help make deploying your application into different environments easier as each environment can define the configuration unique to that environment.
>
> You can remove this block if you do not use environment variables for configuration.

I'm not sure we should encourage people to use filters, as those wouldn't be applied in production scenarios.